blocks/blockoracle_test.go

```diff
@@ -13,13 +13,11 @@ import (
 	"github.com/spacemeshos/go-spacemesh/signing"
 )
 
-var (
-	atxID          = types.ATXID([32]byte{1, 3, 3, 7})
-	nodeID, vrfsgn = generateNodeIDAndSigner()
-	validateVRF    = signing.VRFVerify
-	edSigner       = signing.NewEdSigner()
-	activeSetAtxs  = []types.ATXID{atxID, atxID, atxID, atxID, atxID, atxID, atxID, atxID, atxID, atxID} // 10 ATXs
-)
+var atxID = types.ATXID([32]byte{1, 3, 3, 7})
```

Comment:
i don't see why this block needs to be changed

Reply:
It was auto-formatted in the reverted commit, I'll fix

Reply:
oh thatswhy. i think i am not using gofumpt, so this changes seemed random to me